src/shared_memory_manager.cc

```diff
@@ -406,25 +406,19 @@ SharedMemoryManager::RegisterCUDASharedMemory(
   TRITONSERVER_Error* err =
       OpenCudaIPCRegion(cuda_shm_handle, &mapped_addr, device_id);
   if (err != nullptr) {
+    std::string err_str = TRITONSERVER_ErrorMessage(err);
+    TRITONSERVER_ErrorDelete(err);
```

Comment:
Can this block just be condensed to a `RETURN_IF_ERROR(OpenCudaIPCRegion(...));` instead?